docs/configuration/integrations/github-app-create.md

```diff
@@ -16,6 +16,7 @@ To create the GitHub App:
     | Homepage URL                            | `https://codacy.example.com`                            |
     | User authorization callback URL         | `https://codacy.example.com`                            |
     | Expire user authorization tokens        | Disabled<br/><br/><strong>⚠️ Note:</strong> Currently, Codacy doesn't support <a href="https://docs.github.com/en/developers/apps/building-github-apps/refreshing-user-to-server-access-tokens">expiring user access tokens</a>. Make sure that this option is turned off. |
+    | Request user authorization (OAuth) during installation | Enabled<br/><br/>Make sure this option is selected to request that the installing user grants access to their identity during the installation of your GitHub App. |
```

Comment:
Tiny suggestion just to make it more explicit that the explanation is referring to the Codacy GitHub App being created:

```suggestion
    | Request user authorization (OAuth) during installation | Enabled<br/><br/>Make sure this option is selected to request that the installing user grants access to their identity during the installation of your Codacy GitHub App. |
```